source/_data/MCPubs.bib

```diff
@@ -280,6 +280,9 @@ @InProceedings{nocs:spaa19
   title     = {Near Optimal Coflow Scheduling in Networks},
   year      = {2019},
   pages     = {123--134},
+  publist_link = {paper || nocs-spaa19.pdf},
+  publist_abstract  = {The coflow scheduling problem has emerged as a popular abstraction in the last few years to study data communication problems within a data center. In this basic framework, each coflow has a set of communication demands and the goal is to schedule many coflows in a manner that minimizes the total weighted completion time. A coflow is said to complete when all its communication needs are met. This problem has been extremely well studied for the case of complete bipartite graphs that model a data center with full bisection bandwidth and several approximation algorithms and effective heuristics have been proposed recently. In this work, we study a slightly different model of coflow scheduling in general graphs (to capture traffic between data centers) and develop practical and efficient approximation algorithms for it. Our main result is a randomized 2 approximation algorithm for the single path and free path model, significantly improving prior work. In addition, we demonstrate via extensive experiments that the algorithm is practical, easy to implement and performs well in practice.
+  }
```

Comment:
Could you also add the SPAA'19 conference definition to `publications/index.md` and update this with `public_confkey`?

Reply:
force pushed